index.js

```diff
@@ -22,4 +27,20 @@ DefeatureifyFilter.prototype.processString = function (str) {
 	return defeatureify(str, this.options);
 };
 
+DefeatureifyFilter.prototype.cacheKeyProcessString = function cacheKey(string, relativePath) {
+	return crypto.createHash('md5').update(string, 'utf8').digest('hex');
+};
+
+DefeatureifyFilter.prototype.optionsHash  = function() {
+	if (!this._optionsHash) {
+		this._optionsHash = crypto.createHash('md5').update(stringify(this.options), 'utf8').digest('hex');
+	}
+
+	return this._optionsHash;
+};
```

Comment:
I don't see `optionsHash` anywhere in the code: https://github.com/stefanpenner/broccoli-persistent-filter/blob/125e78ebffb079f72cdfd6e0bb0a9cf9be96cc12/index.js